src/core/fileProcessor.ts

```
@@ -0,0 +1,82 @@
+        }
+
+        if (options.doWrite) {
+            await fs.rename(filePath, newFilePath);
+        }
+
+        return {
+            action: 'rename',
+            newName: newFileName,
+            original: file,
+        };
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)